frontend/src/components/shared/StreamlitMarkdown/styled-components.ts

```diff
@@ -97,16 +99,21 @@ export const StyledLinkIcon = styled.a(({ theme }) => ({
   justifyContent: "center",
 
   // copied from full screen button
-  transition: "opacity 300ms",
+  transform: "scale(0)",
+  transition: "opacity 300ms 100ms, transform 300ms 100ms",
   opacity: 0,
   height: "2.5rem",
   width: "2.5rem",
   zIndex: theme.zIndices.sidebar + 1,
   border: "none",
-  backgroundColor: theme.colors.bgColor,
-  borderRadius: theme.radii.xl,
+  backgroundColor: theme.colors.lightenedBg05,
+  borderRadius: "50%",
 
   svg: {
+    stroke: theme.colors.fadedText60,
+  },
+
+  ":hover svg": {
```

Comment:
Looks like it works, but to keep our normal use, and to match Sass-like language, can be `"&:hover svg"` with the &.